react/Toggle/README.md

```diff
@@ -0,0 +1,40 @@
+### Simplest usage
+
+```js
+<Toggle id="simple-toggle" />
+```
+
+### With a default value
+
+- Toggle is **on** by default:
+
+```js
+<Toggle id="toggle" checked />
+```
+
+- Toggle is **off** by default:
+  
+```js
+<Toggle id="toggle" checked={false} />
+```
+
+### More complex example
+
+```jsx
+<div>
+  <label
+    htmlFor='0'
+    style={{
+      marginRight: '15px',
+      color: state.checked ? 'green' : 'gray'
+    }}
+  >{`Toggle is controlled and ${state.checked
+    ? 'on'
+    : 'off'}`}</label>
+  <Toggle
+    id='0'
+    checked={state.checked}
+    onToggle={() => setState({ checked: !state.checked })}
+  />
+</div>
```

Comment:
I think it's beautiful, more on this on the official styleguidist repository: 

- https://react-styleguidist.js.org/docs/getting-started.html
- https://react-styleguidist.js.org/examples/basic/